### ISSUE TITLE
Invoice sent to wrong customer (generated twice)

### ISSUE THREAD

**Issue by stefanmak** [baselbers/woocommerce-pdf-invoices#52]
On our system (www.zeitschriften-club.de) we generate about 1000 invoices a day and sometimes more than one per minute. It happend a couple of times, that two different customers got the same invoice. So one of them got the correct one and the next/previous order got the same invoice like the one before. Maybe thats the problem because they are generated more or less simultaniously? Would be cool to check that because it makes the site very unserious when you get the wrong invoice with information of somebody else on it!

**Comment by baselbers** [comment]
How much memory do you allocate to WordPress?
I will need more info. Please sent me server logs or errors.

**Comment by stefanmak** [comment]
Hello,

 

we increased the php-memory-size from 512MB to 2GB.

I also changed the time of sending from „new order“ to „order complete“ because here the order number is already generated.

 

Up to now this works, unfortunatly not like before (sending moment would be nice if it would be on „new order“).

 

Best regards

Stefan

 

Von: Bas Elbers [mailto:notifications@github.com] 
Gesendet: Freitag, 25. Dezember 2015 15:18
An: baselbers/woocommerce-pdf-invoices <woocommerce-pdf-invoices@noreply.github.com>
Cc: stefanmak <stefan.mak@gmx.at>
Betreff: Re: [woocommerce-pdf-invoices] Invoice sent to wrong customer (generated twice) (#52)

 

How much memory do you allocate to WordPress?
I will need more info. Please sent me server logs or errors.

**Comment by baselbers** [comment]
The new order email is only for admin purposes and the customer doesn't receives it. Try to disable the new order email and let me know if everything is working fine. This way I know where to look.

**Comment by stefanmak** [comment]
Hi,

 

we increased the php memory limit and set it like you wrote in your mail.

Hopefully it workes well.

 

Best regards

Stefan Mak

 

PS: I noticed another error. When I set the mail to „Finished order“ and you change the order status of more than one customer/order in the woocommerce backend you get a blank page. The debug error is: that the mpdf.php class is already defined

 

Von: Bas Elbers [mailto:notifications@github.com] 
Gesendet: Mittwoch, 13. Jänner 2016 18:27
An: baselbers/woocommerce-pdf-invoices <woocommerce-pdf-invoices@noreply.github.com>
Cc: stefanmak <stefan.mak@gmx.at>
Betreff: Re: [woocommerce-pdf-invoices] Invoice sent to wrong customer (generated twice) (#52)

 

The new order email is only for admin purposes and the customer doesn't receives it. Try to disable the new order email and let me know if everything is working fine. This way I know where to look.
